assets/css/main.scss

```diff
@@ -14,5 +14,19 @@
 }
 
 .author__urls {
-    display: flex;
+  display: flex;
+}
+
+.author__avatar {
+  img {
+    max-width: 180px;
```

Comment:
This will create issues when the page will open on mobile phone. Can this be replace with a relative value or any other mobile friendly method?

Reply:
I'm opened to learn if my assumption here is wrong.